tests/integration/object/util/sns/sns.go

```diff
@@ -0,0 +1,147 @@
+/*
+Copyright 2025 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package sns
+
+import (
+	"context"
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+
+	"github.com/aws/aws-sdk-go-v2/config"
```

Comment:
Note that we are introducing in aws-sdk-go-v2 here.  I felt that it was better to use -v2, rather original flavor as used in the rest of the repo, as this seemed like a lesser evil than increasing the level of effort required for https://github.com/rook/rook/issues/14869